prefect_aws/workers/glue_job_worker.py

```diff
@@ -0,0 +1,156 @@
+import logging
+import time
+from typing import Any, Optional
+
+import anyio
+from prefect.server.schemas.core import FlowRun
+from prefect.utilities.asyncutils import run_sync_in_worker_thread
+from prefect.workers.base import (
+    BaseJobConfiguration,
+    BaseVariables,
+    BaseWorker,
+    BaseWorkerResult,
+)
+from pydantic import Field
```

Comment:
We recently added support for `pydantic` V2, so adding this check should fix the tests.

```suggestion
if PYDANTIC_VERSION.startswith("2."):
    from pydantic.v1 import Field
else:
    from pydantic import Field
```

Reply:
Thanks, fix it
https://github.com/PrefectHQ/prefect-aws/pull/308/commits/bae5e121a5bf2bb3bb6da0eead9821fb03323280
https://github.com/PrefectHQ/prefect-aws/pull/308/commits/3b501e2207da342c073bbe46dca95cd8e0e740a8